lib/middleware/websocket/interactors/update_all_clients.rb

```diff
@@ -0,0 +1,21 @@
+require './lib/middleware/websocket/interactors/build_payload'
+
+module Websocket
+  module Interactor
+    class UpdateAllClients
+      def initialize(clients: clients)
+        @clients = clients
+        @payload = generate_payload
+        perform_update
```

Comment:
We should try out best to not do any work that has "side-effects" in our `initialize` calls.

Reply:
@swcraig I agree, just needed to get back into the swing of Ruby. I was basically treating the initialize as a `call` method; this is updated now.